ddev/hatch.toml

```diff
@@ -14,3 +14,7 @@ dependencies = [
 pre-install-commands = [
   "python -m pip install --disable-pip-version-check {verbosity:flag:-1} -e ../datadog_checks_dev[cli]",
 ]
+env-exclude = [
+  "DD_GITHUB_USER",  # Used in `ddev config show` tests
+  "DD_SITE",  # Used in `ddev config show` tests
```

Comment:
There are more, at least I've found these:
https://github.com/DataDog/integrations-core/blob/82f574e057b6d5f7daa4715d7e3e70ff952c3988/ddev/src/ddev/config/model.py#L195-L199

https://github.com/DataDog/integrations-core/blob/82f574e057b6d5f7daa4715d7e3e70ff952c3988/ddev/src/ddev/config/model.py#L469

Reply:
I switched to 'DD_*'.

Regarding https://github.com/DataDog/integrations-core/pull/13672#issuecomment-1378457646, I think this is what we should do if we want to mock them in a specific test, but here we want to make sure our own personal config does not leak into the tests. Not sure that makes sense, what do you think?

Reply:
The way I see it is that if the test is not explicit about what the environment has to look like (as far as the code under test is concerned), then the outcome of the test is still depending on the environment. In other words, the tests now depend on hatch and the hatch config to work in a predictable way.

Reply:
If this means we have to comb through the ddev tests, I'd be up for that, I think it would improve the test suite and the code.

Reply:
I see your point. What do you think of this approach then? https://github.com/DataDog/integrations-core/pull/13672/commits/a07171416bda0a66f98540c08f84698e966ba414

Reply:
I think it'll do :)